index.js

```diff
@@ -4,7 +4,28 @@
  * Depending on the operation, either add up all of the numbers or subtract all of the numbers, from left to right.
  * @returns {number} The result of either adding all numbers or subtracting all numbers, depending on the arguments added to the command line.
  */
-function calculator() {}
-
+ function calculator() {
+    let result = 0 ;
+    if (process.argv[2] !== 'plus' && process.argv[2] !== 'minus') {
+     result = "Invalid operation: " + process.argv[2];
+  } if (process.argv.length === 3) {
+     result = "No numbers provided...";
+  } if (process.argv.length === 2) {
```

Comment:
If i run `node run.js 1 2 3`, I get `Invalid operation: 1`. How can you update this conditional to make sure you get the correct error message?